src/components/LoadingBar/LoadingBar.tsx

```diff
@@ -0,0 +1,72 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { merge } from '@utilities/merge';
+import React from 'react';
```

Comment:
The only thing I would change here is move the 'react' import to the top.

Reply:
fixed